src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -131,6 +131,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.DeleteFile.cs">
       <Link>Common\Interop\Windows\Interop.DeleteFile.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.EncryptDecrypt.cs">
```

Comment:
This needs to move down to the Win32 only section to avoid failing the WinRT/UWP build.

Reply:
thanks